Code/skyrim_ui/src/app/components/connect/connect.component.html

```diff
@@ -20,10 +20,10 @@
     (keydown.enter)="connect()"
   >
   <span class="save-password">
-    <app-checkbox
+    <input type="checkbox"
```

Comment:
put it in a new line for consistency
```suggestion
    <input
      type="checkbox"
```